package.json

```diff
@@ -82,14 +82,16 @@
     },
     "dependencies": {
         "feedparser": "^2.2.9",
+        "fs-extra": "^8.1.0",
```

Comment:
What's the use of `fs-extra` here? Because at the moment, it looks like we're adding a 47.14kb package to the launcher just to promisify a single method - which is what the built-in `util.promisify` function is for.

Reply:
Currently, I'm using it for the `fs.move` method. As I'm not really sure if `fs.rename` works well, is it a decent option though?

Reply:
I didn't look into this a ton, but it seems that fs-extra works even between different partitions: https://stackoverflow.com/questions/4568689/how-do-i-move-file-a-to-a-different-partition-or-device-in-node-js
whereas plain fs whould need bunch of extra code to handle that. So fs-extra use should be kept.